website/content/search.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Search doesn't work properly for me in the preview site.

Reply:
I only want to hook up search when we're ready to publicize this since it uses Google Custom Search so would likely initiate the indexing.